code/modules/projectiles/guns/energy/laser.dm

```diff
@@ -36,7 +36,7 @@ obj/item/weapon/gun/energy/retro
 	name = "NT LG \"Destiny\""
 	icon_state = "caplaser"
 	item_state = "caplaser"
-	desc = "This weapon is old, yet still robust and reliable. Its marked with old Nanotrasen brand, like a last reminder of what this corporation was, before church take control of everything."
+	desc = "This weapon is old, yet still robust and reliable. It's marked with old Nanotrasen brand, like a last reminder of what this corporation was, before the Church take control of everything."
```

Comment:
Improved but not perfect, try this

"This weapon is old, yet still robust and reliable. It's marked with an old Nanotrasen brand, a distant reminder of what this corporation was before the Church took control of everything."